server/storeProfile.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+require_once 'config.php';
+
+storeProfile(_$POST);
+
+function storeProfile($data) {
+  GLOBAL $DB_CONN;
+  $response = array();
+  $response['success'] = false;
+
+  $DB_CONN -> beginTransaction();
+
+  try {
+    $clientInsert = "INSERT INTO vita.user
+    (
+      firstName,
+      lastName,
+      phone,
+      email
+    )
+    VALUES
+    (
+      ?,
+      ?,
+      ?,
+      ?
+    );";
+    $userparams = array(
+      $data['firstName'],
+      $data['lastName'],
+      $data['email'],
+      $data['phone']
+    );
+    $stmt = $DB_CONN -> prepare($userInsert)
+    $stmt -> execute($userParams);
+
+    $userabilityID = $DB_CONN -> lastInsterId();
```

Comment:
Don't capitalize the D in ID. And lower camelcase again. So this should read `$userAbilityId`. Also, you spelled `lastInsertId` wrong. It's clear this file has never been tested, otherwise errors like these would've been caught (Ik this is a WIP, but I expected this to be done).